app/models/qdm/basetypes/data_element.rb

```diff
@@ -66,10 +66,18 @@ def shift_dates(seconds)
         end
 
         # Special case for facility locations
-        next unless field == 'facilityLocations'
-        send(field).each do |facility_location|
-          facility_location['locationPeriod'][:low] = (facility_location['locationPeriod'][:low].to_time + seconds).to_datetime
-          facility_location['locationPeriod'][:high] = (facility_location['locationPeriod'][:high].to_time + seconds).to_datetime
+        if field == 'facilityLocations'
+          send(field).each do |facility_location|
+            facility_location['locationPeriod'][:low] = (facility_location['locationPeriod'][:low].to_time + seconds).to_datetime
```

Comment:
These 2 lines are duplicated below and should be pulled out into a separate function.